provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -0,0 +1,89 @@
+# azure-iot-provisioning-http Requirements
+
+## Overview
+This module provides HTTP protocol support to communicate with the Azure device provisioning service
+
+```js
+'use strict';
+
+export class Http extends EventEmitter implements DeviceProvisioningTransport {
+  api-version: string;
+  operationPollingIntervalInSeconds: Number
+  constructor(idScope: string, registrationId: string) : void;
+  register(authorization?: string, forceRegistration?: boolean, done(err: Error, result?: Any) => void) : void;
+}
+```
+
+## Public Interface
+
+### constructor(idScope: string, registrationId: string) : void
+The `constructor` creates an Http transport object used to communicate with the Azure device provisioning service
+
+**SRS_NODE_PROVISIONING_HTTP_18_001: [** The `Http` constructor shall accept the following properties:
+- `idScope` - a string specifiying the scope of the provisioning operations,
+- `registrationId` - the registration id for the specific device 
+**]**
+
+
+### register(authorization?: string, forceRegistration?: boolean, callback (err: Error, result?: Any) => void) : void;
```

Comment:
>callback [](start = 66, length = 8)

missing ':'. also you can't have a mandatory parameter after optional parameters